index.js

```diff
@@ -43,37 +43,57 @@ const cardsContent = [
   },
 
 ];
+
 const divParent = document.querySelector('.speak-container');
 const dynamicSpeakersCards = () => {
   let speakersDiv = '';
 
-  for (let i = 0; i < cardsContent.length; i += 1) {
-    const speakersCard = `
-    <div class="speaker ${cardsContent[i].cardId}">
-      <div class="speaker-img">
-        <img src="${cardsContent[i].picture}" alt="" />
-      </div>
-      <div class="speaker-container">
-        <h2 class="speaker-name">${cardsContent[i].name}</h3>
-        <p class="speaker-title">${cardsContent[i].title}</p>
-        <p class="speaker-info">${cardsContent[i].description}</p>
-      </div>
-    </div>`;
-    speakersDiv += speakersCard;
+  if (divParent !== null) {
+    for (let i = 0; i < cardsContent.length; i += 1) {
+      const speakersCard = `
+      <div class="speaker ${cardsContent[i].cardId}">
+        <div class="speaker-img">
+          <img src="${cardsContent[i].picture}" alt="" />
+        </div>
+        <div class="speaker-container">
+          <h2 class="speaker-name">${cardsContent[i].name}</h3>
+          <p class="speaker-title">${cardsContent[i].title}</p>
+          <p class="speaker-info">${cardsContent[i].description}</p>
+        </div>
+      </div>`;
+      speakersDiv += speakersCard;
+    }
+    divParent.innerHTML = speakersDiv;
   }
-  divParent.innerHTML = speakersDiv;
 };
 dynamicSpeakersCards();
 
 /* Click See more */
 const show = document.getElementById('btnshow');
 
-show.addEventListener('click', () => {
-  divParent.classList.toggle('morebtn');
-  /* Change content of button */
-  if (show.innerHTML === 'SHOW MORE') {
-    show.innerHTML = 'SHOW LESS';
-  } else if (show.innerHTML === 'SHOW LESS') {
-    show.innerHTML = 'SHOW MORE';
-  }
-});
\ No newline at end of file
+if (show !== null) {
+  show.addEventListener('click', () => {
+    divParent.classList.toggle('morebtn');
+    /* Change content of button */
+    if (show.innerHTML === 'SHOW MORE') {
+      show.innerHTML = 'SHOW LESS';
+    } else if (show.innerHTML === 'SHOW LESS') {
+      show.innerHTML = 'SHOW MORE';
+    }
+  });
+}
+
+/* Mobile nav-bar */
+const menuHamburguer = document.querySelector('.hamburger');
+const myNav = document.querySelector('.items');
+
+menuHamburguer.addEventListener('click', () => {
+  menuHamburguer.classList.toggle('active');
+  myNav.classList.toggle('active');
+});
+
+/* clicking in every item of the menu, the menu desapear */
+document.querySelectorAll('.item').forEach((n) => n.addEventListener('click', () => {
+  menuHamburguer.classList.remove('active');
+  myNav.classList.remove('active');
```

Comment:
- [ ] As requested by the previous reviewer, the menu should fill the entire screen. To enhance your UI design, it would be preferable to make your menu cover the entire screen.
- Your design:
<img width="205" alt="image" src="https://user-images.githubusercontent.com/77190203/183141193-e0ae581d-801a-428a-8aa6-a9a730d3f672.png">

Reply:
Done. Thanks!